requirements-dev.txt

```diff
@@ -2,23 +2,18 @@
 -r requirements.txt
 
 # test
-pytest==3.5.1
-pytest-pycodestyle==1.4.0
-pytest-cov==2.5.1
-pytest-twisted==1.6
-pytest_localserver==0.4.1
-pytest_sanic==0.1.12
-treq==17.8.0
+pytest==4.5.0
+pytest-cov==2.7.1
+pytest-localserver==0.5.0
+pytest-sanic==1.0.0
+freezegun==0.3.12
 responses==0.9.0
-httpretty==0.9.5
-freezegun==0.3.10
 nbsphinx==0.3.2
-aioresponses==0.5.2
-mock==2.0.0
+aioresponses==0.6.0
 moto==1.3.8
 
 # pipeline dependencies
-spacy==2.0.18
+spacy==2.1.4
```

Comment:
should we use sth like https://dependabot.com/ to be notified as soon as there are new versions instead of doing this manually?

Reply:
looked into it a while ago, as far as I know github bought them so it is most likely going to be a GH feature soon anyways.

Reply:
yeah, they talked about it on the conference: https://dependabot.com/blog/hello-github/